central/alert/mappings/options.go

```diff
@@ -7,13 +7,17 @@ import (
 	"github.com/stackrox/rox/pkg/search"
 )
 
-// TODO: [ROX-10206] Reconcile storage.ListAlert search terms with storage.Alert
-
 // OptionsMap is exposed for e2e test.
-var OptionsMap search.OptionsMap
+var (
+	OptionsMap      search.OptionsMap
+	optionsToRemove = [...]search.FieldLabel{"SORT_Lifecycle Stage", "SORT_Enforcement"}
+)
 
 func init() {
-	OptionsMap = search.Walk(v1.SearchCategory_ALERTS, "list_alert", (*storage.ListAlert)(nil))
+	OptionsMap = search.Walk(v1.SearchCategory_ALERTS, "alert", (*storage.Alert)(nil))
+	for _, opt := range optionsToRemove {
+		OptionsMap.Remove(opt)
+	}
 	alertOptions := schema.AlertsSchema.OptionsMap.Clone()
```

Comment:
I think we don't need this and the loop after this because we have already set the OptionsMap to the desired state.

Reply:
Good catch.